.changeset/short-planes-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/project-access': patch
```

Comment:
@PetrStribrny as mentioned, I think this is minor as the result of call `getProject()` as more data. 
```suggestion
'@sap-ux/project-access': minor
```

Reply:
> @PetrStribrny as mentioned, I think this is minor as the result of call `getProject()` as more data.

@Klaus-Keller done